tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1080,6 +1080,10 @@ def resize_image_with_crop_or_pad(image, target_height, target_width):
       of shape `[height, width, channels]`.
     target_height: Target height.
     target_width: Target width.
+    mode: One of "CONSTANT", "REFLECT", or "SYMMETRIC" (case-insensitive)
```

Comment:
Could we please append a description to `mode` pointing users to the description / example in `pad_to_bounding_box`? e.g. "See examples in `pad_to_bounding_box`."

Reply:
I have included a line pointing users to find description and examples in 'pad_to_bounding_box'